<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NumberPhone</class>
 <widget class="QWidget" name="NumberPhone">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Номер телефона</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>411</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>Введите номер мобильного телефона клиента:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNumPhone">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSaveNumPhone">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBack4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>440</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
